spec/test_helper.rb

```diff
@@ -15,3 +16,11 @@ def deliverable(options = {})
     mail(options)
   end
 end
+
+def fixture_for(name, type:)
+  File.read(File.join('spec', 'fixtures', type.to_s, "#{name}.#{type}"))
+end
+
+RSpec.configure do |config|
+  config.use_transactional_fixtures = true
```

Comment:
Why is this necessary? (Just wondering)

Reply:
because entries are updated/created and they needed to be cleared out

Reply:
OG gem had a before to destroy all inbound emails